model/k8s/registry.yaml

```
@@ -687,3 +687,74 @@ groups:
           [Kubernetes Resource Quotas documentation](https://kubernetes.io/docs/concepts/policy/resource-quotas/#object-count-quota)
           for more details.
         examples: [ 'count/replicationcontrollers' ]
+      - id: k8s.pod.status.reason
+        type:
+          members:
+            - id: evicted
+              value: 'Evicted'
+              brief: 'The pod is evicted.'
+              stability: experimental
+            - id: node_affinity
+              value: 'NodeAffinity'
+              brief: 'The pod is in a status because of its node affinity'
+              stability: experimental
+            - id: node_lost
+              value: 'NodeLost'
+              brief: >
+                The reason on a pod when its state cannot be confirmed as kubelet is unresponsive
+                on the node it is (was) running.
+              stability: experimental
+            - id: shutdown
+              value: 'Shutdown'
+              brief: 'The pod is shutdown'
```

```suggestion
              brief: 'The node is shutdown'
```
see https://github.com/kubernetes/kubernetes/blob/bedb915a4e7120c90fee1343693a9a403cb06d92/pkg/kubelet/events/event.go#L71